src/website/shared/management/commands/update_groups.py

```diff
@@ -0,0 +1,62 @@
+import argparse
+import logging
+from typing import Any
+
+from django.conf import settings
+from django.contrib.auth.models import Group, User
+from django.core.management.base import BaseCommand
+from django.db import transaction
+from shared.auth import get_team_member_ids
+
+logger = logging.getLogger(__name__)
+
+
+class Command(BaseCommand):
```

Comment:
We probably rather want to set up a webhook on [team membership events](https://docs.github.com/en/webhooks/webhook-events-and-payloads#membership)